api/proto/info/info.proto

```diff
@@ -783,3 +791,11 @@ message Capabilities {
   // I/O Virtualization support
   bool IOVirtualization = 3;
 }
+
+// AppMetaData sends meta data of an application instance
+// e.g kubeconfig, etc
+message ZInfoAppMetaData {
+  string uuid = 1;
+  AppMetaDataType type = 2;
+  repeated bytes data = 3;
```

Comment:
Is the intent to have one datablob of bytes or multiple?
repeated means multiple arrays of bytes.

Reply:
Also, if would make sense to document in the comment whatever size restriction we put. AFAIK the Zededa controller currently limits info messages to 32/64kB, so the AppMetaData would inherit that limit. But it probably makes sense to put an explicitly defined size limit in the proto file in this case since otherwise the size an application can send could be different for different controller implementations.